prog/weaveproxy/main.go

```diff
@@ -57,6 +59,10 @@ func main() {
 	Log.Infoln("weave proxy", version)
 	Log.Infoln("Command line arguments:", strings.Join(os.Args[1:], " "))
 
+	if withDNS {
+		Log.Warning("--with-dns option has been removed; DNS is on by default")
```

Comment:
No, I am not talking about `weave run` but the `--with-dns` case you added to `deprecation_warnings` in this very PR.